app/utils/cli-install.js

```diff
@@ -0,0 +1,50 @@
+const {cliScriptPath} = require('../config/paths');
+
+const pify = require('pify');
+const fs = require('fs');
+
+const lstat = pify(fs.lstat);
+const readlink = pify(fs.readlink);
+const unlink = pify(fs.unlink);
+const symlink = pify(fs.symlink);
+
+const target = process.platform === 'darwin' ? '/usr/local/bin/hyper' : '/usr/bin/hyper';
+const source = cliScriptPath;
+
+const checkInstall = () => {
```

Comment:
I would write all these promise-chain-functions without the return statement (implicit return) saves a few lines and also makes the code more readable, but if it's like this in the readme, keep it. 👍  (very very minor thing 😂 )

Reply:
I'll change that

Reply:
I really prefer a first explicit return in these functions